fastlane/lib/fastlane/actions/add_git_tag.rb

```diff
@@ -14,7 +14,7 @@ def self.run(options)
         cmd << ["-am #{message.shellescape}"]
         cmd << '--force' if options[:force]
         cmd << '-s' if options[:sign]
-        cmd << "'#{tag}'"
+        cmd << tag.shellescape
```

Comment:
this changes the behavior a bit as it doesn't wrap the param in `'` any more, but as the tag itself is now `shellescape`d this should not cause any problem.